go/common/crypto/signature/signer.go

```diff
@@ -111,6 +120,19 @@ func UnsafeResetChainContext() {
 	chainContext = Context("")
 }
 
+// UnsafeAllowUnregisteredContexts bypasses the context registration check.
+//
+// This function is only for the benefit of implementing a remote signer.
+func UnsafeAllowUnregisteredContexts() {
+	allowUnregisteredContexts = true
+}
+
+// IsUnsafeUnregisteredContextsAllowed returns true iff context registration
+// checks are bypassed.
+func IsUnsafeUnregisteredContextsAllowed() bool {
```

Comment:
Maybe add a trivial test for the two functions in `signer_test.go`.

Reply:
Fixed.